### PR TITLE
Add striped bg to pending payment section header

### PR DESCRIPTION
**Pull request by buoyad**
r? @keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
might be better to make this a common adapter

**Comment by buoyad** [comment]
@chrisnojima Done

File: shared/common-adapters/background-repeat-box/index.native.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+const styles = Styles.styleSheetCreate({
+  backgroundImage: {...Styles.globalStyles.fillAbsolute, height: 'auto', width: 'auto'},
+  container: {
+    position: 'relative',
+  },
+})
+
+export default SectionHeader
```
Comment (chrisnojima) [inline shared/common-adapters/background-repeat-box/index.native.js]: should be BackgroundRepeatBox

Comment (buoyad) [reply shared/common-adapters/background-repeat-box/index.native.js]: whoops, thanks

**Pull request merged**